app/models/user.rb

```diff
@@ -16,6 +18,10 @@ class User < ApplicationRecord
 
   attr_accessor :remember_token, :activation_token, :reset_token
 
+  def feed
+    microposts
```

Comment:
```suggestion
    microposts.newest
```
ủa em tạo scope newest mà ko dùng thì hơi phí nha